src/pseudopeople/configuration.py

```diff
@@ -80,6 +80,8 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
 
         # Loop through columns and their applicable column noise types
         for column in form.columns:
+            if not column.noise_types:
+                continue
```

Comment:
This is unnecessary. It doesn't save any meaningful time and adds two lines of code. It's fine to loop over an empty `column.noise_types` tuple.